src/util/ToJson.cpp

```diff
@@ -80,7 +83,9 @@ folly::dynamic toJson(const meta::cpp2::ColumnDef &column) {
         obj.insert("nullable", folly::to<std::string>(*column.get_nullable()));
     }
     if (column.__isset.default_value) {
-        obj.insert("defaultValue", column.get_default_value()->toString());
+        graph::QueryExpressionContext ctx;
+        auto value = Expression::decode(*column.get_default_value())->eval(ctx(nullptr));
+        obj.insert("defaultValue", value.toString());
```

Comment:
It's not necessary to update here. as you say, we should response to user the `now()` string rather than current timestamp.

Reply:
good catch